src/pages/tasks/TaskAssigneeSelectorModal.tsx

```diff
@@ -131,11 +100,11 @@ function TaskAssigneeSelectorModal(props) {
     };
 
     const report = useMemo(() => {
-        if (!props.route.params || !props.route.params.reportID) {
+        if (!route.params?.reportID || !reports) {
             return null;
         }
-        return props.reports[`${ONYXKEYS.COLLECTION.REPORT}${props.route.params.reportID}`];
-    }, [props.reports, props.route.params]);
+        return reports[`${ONYXKEYS.COLLECTION.REPORT}${route.params.reportID}`];
```

Comment:
Can this be moved to `withOnyx` hoc? Looks like this work (_getting_) we usually do outside of a component.